### PR TITLE
Fixing default event location URL for dynamic events

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

https://www.loom.com/share/ef27b7edb2554fbf934ec80aa01d8df9

**Environment**: Staging(main branch) / Production

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
- [ ] Set a default location other than cal.video (Link apps)
- [ ] Create a booking

### CODE REVIEW

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 1]
```diff
@@ -794,7 +794,11 @@ async function handler(
   const seed = `${organizerUser.username}:${dayjs(reqBody.start).utc().format()}:${new Date().getTime()}`;
   const uid = translator.fromUUID(uuidv5(seed, uuidv5.URL));
 
-  const bookingLocation = getLocationValueForDB(locationBodyString, eventType.locations);
+  let bookingLocation = getLocationValueForDB(locationBodyString, eventType.locations);
```
Comment (sean-brydon) [inline packages/features/bookings/lib/handleNewBooking.ts:797]: getLocationValueForDB returns locationBodyString if no location is found in the event type

File: packages/features/bookings/lib/handleNewBooking.ts [hunk 2]
```diff
@@ -794,7 +794,11 @@ async function handler(
...
+
+  if (dynamicUserList.length > 1 && defaultLocationUrl) {
+    bookingLocation = defaultLocationUrl;
+  }
```
Comment (sean-brydon) [inline packages/features/bookings/lib/handleNewBooking.ts:801]: So if there dynamic users -> re set the bookingLocation to the defaultLocationUrl which is fetched above

File: packages/features/bookings/lib/handleNewBooking.ts (hunk 1 above)
Comment (hariombalhara) [inline packages/features/bookings/lib/handleNewBooking.ts:797]: I think we would be able to co-locate location calculation related code like this
Screenshot 2023-04-20 at 4 09 10 PM

Comment (sean-brydon) [reply packages/features/bookings/lib/handleNewBooking.ts:797]: I see! Good shout! Just having lunch will implement this 🙏

Comment (hariombalhara) [reply packages/features/bookings/lib/handleNewBooking.ts:797]: Also, at line 782 in my screenshot, why don't we directly set locationBodyString equal to the defaultLocationUrl. THen we won't need the below new code.

Comment (sean-brydon) [reply packages/features/bookings/lib/handleNewBooking.ts:797]: Yeah just testing and that was the only issue it seems

**Review by hariombalhara — APPROVED** [review]

**Pull request merged**